### ISSUE TITLE
Issues with importing MOM files with sanctum of twilight expansion

### ISSUE THREAD

**Issue by TamsynShute** [NPBruce/valkyrie#797]
Importing the new sanctum twilight files moves all the token placement and token names. Also "?" and "!" tokens are now missing. Qitems state multiple languages in item descriptions.

**Comment by NPBruce** [comment]
Yes, Importing from MoM 1.5 will be broken, though I wasn't aware of QItem changes, I will investigate.

This will be completed as part of #728 .

**Comment by NPBruce** [comment]
To clarify, where are you seeing the multiple languages?  Can you provide a screen shot?

**Comment by TamsynShute** [comment]
![screenshot_20180418-232333](https://user-images.githubusercontent.com/38496588/38970035-4b81fb26-4361-11e8-962c-89e4f506bd5a.png)
![screenshot_20180418-232326](https://user-images.githubusercontent.com/38496588/38970040-4f2e89d8-4361-11e8-8621-3f754d7b8be7.png)
![screenshot_20180418-232316](https://user-images.githubusercontent.com/38496588/38970047-53d34ed8-4361-11e8-9e2d-0caab758e885.png)
![screenshot_20180418-232305](https://user-images.githubusercontent.com/38496588/38970056-57c555e0-4361-11e8-8696-989c72120770.png)

**Comment by TamsynShute** [comment]
![screenshot_20180418-234105](https://user-images.githubusercontent.com/38496588/38970405-4cb485d4-4363-11e8-8eab-ae5433343ffe.png)
![screenshot_20180418-234053](https://user-images.githubusercontent.com/38496588/38970407-4eb34f64-4363-11e8-9673-9e4499d42670.png)
![screenshot_20180418-234046](https://user-images.githubusercontent.com/38496588/38970411-52dbe1f0-4363-11e8-828e-2644cda8b0d7.png)
![screenshot_20180418-234036](https://user-images.githubusercontent.com/38496588/38970414-55b8c37a-4363-11e8-957c-924c3942b55c.png)

**Comment by redwolf2** [comment]
Can confirm. This is 1.6.1 with the latest official Mom version. This will also happen with the latest development build.
![text_bug](https://user-images.githubusercontent.com/4141621/38971004-18ed77f2-4399-11e8-8942-5e59a45d45d0.png)
